File: elem/num_sum.js
const readline = require("readline");
const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout 
}); 

const num_sum = num => {
    let result = [];
    for (let x = 1; x <= num; x++) {
        result.push(x);
    }
    return result.reduce((a, b) => a += b);
}

const hey_there = () => {
    let num;
    rl.question('Enter a number between 1 and 99 ', function(n) {
        let temp = num_sum(+n)
        console.log(`Your sum is ${temp}`);
        rl.close();
    })
}

rl.on('close', function() {
    process.exit(0);
});

hey_there();

File: elem/num_sum.py


def num_sum(num):
    r = range(1, num + 1)
    result = 0
    for x in r:
        if x % 3 == 0 or x % 5 == 0:
            result += x 
    
    return result 


def hey_there():
    print('Enter a number between 1 and 99')
    num = raw_input()
    summed = num_sum(int(num))
    print('Your sum is {}'.format(summed))


hey_there()